### ISSUE TITLE
Scrolling using arrow keys

### ISSUE THREAD

**Issue by rbrecheisen** [ivmartel/dwv#571]
Hi,

I'm very impressed with the quality of your DWV tool but I was wondering whether it's possible to go through the slices step-by-step just using the arrow keys. Touch is not ideal if you are searching for a very specific slice. Also, the displays tends to jitter sometimes when you use touch-based scrolling.

Thanks,

Ralph

**Comment by ivmartel** [comment]
Thanks. You can scroll with `Ctrl + arrow`.

**Comment by arturojain** [comment]
I see `ctrl + arrow left/right` on macOS changes spaces too.

**Comment by arturojain** [comment]
Is there a function we can call to create our own keyboard bindings?

**Comment by ivmartel** [comment]
Yes, since v0.27.0 it is up to the client to set the event handlers. dwv provides a default [onKeyDown](https://github.com/ivmartel/dwv/blob/v0.27.0/src/app/application.js#L759) that you can use or you can set your own. Check out the demo apps to see how this is done.
